src/javascript/app_2/App/Components/Elements/FullPageModal/full_page_modal.jsx

```diff
@@ -0,0 +1,51 @@
+import React       from 'react';
+import PropTypes   from 'prop-types';
+
+const FullPageModal = ({
+    body,
+    cancel_button_text,
+    confirm_button_text,
+    onConfirm,
+    onCancel,
+    show,
+    title,
+}) => {
+    if (show) {
+        return (
+            <div className='full-page-modal'>
+                <div className='modal-dialog'>
+                    <h1>{title}</h1>
+                    <p>{body}</p>
+                    <div className='modal-footer'>
+                        <div
+                            className='btn flat effect primary'
+                            onClick={onCancel}
+                        >
+                            <span>{cancel_button_text}</span>
+                        </div>
+                        <div
+                            className='btn flat effect primary'
+                            onClick={() => onConfirm()}
```

Comment:
@easteregg 
Since the `onCofirm` is a function and doesn't need to pass any parameters to it, it's sufficient to assign its name to `onClick`;